cpp/ql/src/Summary/LinesOfCode.ql

```diff
@@ -0,0 +1,11 @@
+/**
+ * @id cpp/summary/lines-of-code
+ * @name Total lines of C/C++ code in the database.
```

Comment:
```suggestion
 * @name Total lines of C/C++ code in the database
```

[The name should not include a full stop](https://github.com/github/codeql/blob/main/docs/query-metadata-style-guide.md#query-name-name).